src/test/java/EnergyTest.java

```diff
@@ -0,0 +1,38 @@
+import imageData.Energy;
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EnergyTest {
+
+    @Test
+    void testBuildEnergyArray() {
```

Comment:
The name of the test method shouldn't start with the word "test.

Reply:
The name of the test method should correspond to the name of the method being tested.

Reply:
You should use `given-when-then` comments so that people can figure out what the parts of your test method are.